ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -326,7 +332,9 @@ bool CurlNetworking::sendRequest(shared_ptr<CurlRequest> curlRequest, folly::dyn
   return status;
 } 
 bool CurlNetworking::abortRequest(shared_ptr<CurlRequest> curlRequest) {
+  std::scoped_lock lock(mutex_);
   if(curlRequest->handle) {
```

Comment:
will curlRequest always be valid or it's validity to be checked before deferring. My suggestion is to check & defer from the programatical point